docs/guides/void-linux/_include/live-environment.rst

```diff
@@ -0,0 +1,19 @@
+Configure Live Environment
+--------------------------
+
+Generate ``/etc/hostid``
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block::
+
+  zgenhostid
+
+Source ``/etc/os-release``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
```suggestion

The file `/etc/os-release` defines variables that describe the running distribution. In particular, the `$ID` variable defined within can be used as a short name for the filesystem that will hold this installation.

```

Reply:
created `_include/os-release.rst` with the suggested text